src/utils/form-page-utils.ts

```diff
@@ -0,0 +1,14 @@
+import { FormSection } from 'src/types';
+import { isTrue } from './boolean-utils';
+
+export function isSectionExpanded(section: FormSection, isFormExpanded): boolean {
+  let sectionIsExpanded = true;
+
+  if (isFormExpanded !== undefined) {
+    sectionIsExpanded = isFormExpanded;
+  }
+  if (!isTrue(section?.isExpanded)) {
+    if (section?.isExpanded !== undefined && section?.isExpanded !== null) sectionIsExpanded = false;
+  }
+  return sectionIsExpanded;
+}
```

Comment:
```suggestion
export function isSectionExpanded(section: FormSection, isFormExpanded: boolean): boolean {
  if (isFormExpanded !== undefined) {
    return isFormExpanded;
  }
  
  if (section?.isExpanded !== undefined && section?.isExpanded !== null) {
    return isTrue(section.isExpanded);
  }

  return true;
}
```

Does this work?

Reply:
Might we want to run checks for both conditions? ie: a scenario where `isFormExpanded` evaluates to `true` but a certain section is explicitly marked to collapse?

Reply:
Yeah, I guess so.

Reply:
Since `isFormExpanded` state is managed by the workspace toggle, I would expect it to take precedence. I seem to like @denniskigen's suggested stub!